exchanges/coinbasepro/coinbasepro_wrapper.go

```diff
@@ -283,23 +283,23 @@ func (c *CoinbasePro) Run() {
 }
 
 // FetchTradablePairs returns a list of the exchanges tradable pairs
-func (c *CoinbasePro) FetchTradablePairs(ctx context.Context, asset asset.Item) ([]string, error) {
-	pairs, err := c.GetProducts(ctx)
+func (c *CoinbasePro) FetchTradablePairs(ctx context.Context, a asset.Item) (currency.Pairs, error) {
+	products, err := c.GetProducts(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	format, err := c.GetPairFormat(asset, false)
-	if err != nil {
-		return nil, err
-	}
-
-	products := make([]string, len(pairs))
-	for x := range pairs {
-		products[x] = pairs[x].BaseCurrency + format.Delimiter + pairs[x].QuoteCurrency
+	pairs := make([]currency.Pair, len(products))
+	for x := range products {
+		var pair currency.Pair
+		pair, err = currency.NewPairFromStrings(products[x].BaseCurrency,
```

Comment:
Final nit, noticed a collision with:

```
{"id":"BTCAUCTION-USD","base_currency":"BTC","quote_currency":"USD","quote_increment":"0.01","base_increment":"0.00000001","display_name":"BTCAUCTION/USD","min_market_funds":"1","margin_enabled":false,"post_only":false,"limit_only":false,"cancel_only":true,"status":"online","status_message":"auction ended","trading_disabled":false,"fx_stablecoin":false,"max_slippage_percentage":"0.02000000","auction_mode":false}
```

It uses the same base/quote currencies as BTC regular. We can instead use the display name but there's a bug in there currently: https://github.com/thrasher-corp/gocryptotrader/blob/master/exchanges/coinbasepro/coinbasepro_types.go#L17 👀